cmd/nerdctl/images.go

```diff
@@ -59,14 +59,15 @@ Properties:
 - BLOB SIZE:  Size of the blobs (such as layer tarballs) in the content store
 `
 	var imagesCommand = &cobra.Command{
-		Use:               "images",
-		Short:             shortHelp,
-		Long:              longHelp,
-		Args:              cobra.MaximumNArgs(1),
-		RunE:              imagesAction,
-		ValidArgsFunction: imagesShellComplete,
-		SilenceUsage:      true,
-		SilenceErrors:     true,
+		Use:                   "images [FLAGS] [REPOSITORY[:TAG]]",
```

Comment:
should we make it lower case? e.g. `[flags]`. It would be consistent with other commands.

Reply:
Good point, updated in [14ece4f](https://github.com/davidhsingyuchen/nerdctl/pull/1/commits/14ece4fb13a640ef78081e7cc629aaccdc5d2568)